cli/config/init.entrypoint.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { DataQuery } from '@dhis2/app-runtime'
 import i18n from '@dhis2/d2-i18n'
+import classes from './init.App.module.css'
```

Comment:
This needs to be `./App.module.css`, it's renamed when the template is actually installed

Reply:
Thanks for spotting that. I'm pretty sure I typed the file name without `init.`. I suspect my VSCode import plugin has tried to correct me 🤦  .... Anyhow, it's corrected now.